src/pyodide/python-entrypoint-helper.js

```diff
@@ -61,10 +61,19 @@ function disabledLoadPackage() {
   throw new Error("We only use loadPackage in workerd");
 }
 
-async function setupPackages(pyodide) {
-  patchLoadPackage(pyodide);
+const canonicalizeNameRegex = /[-_.]+/g;
+
+/**
+ * Normalize a package name. Port of Python's packaging.utils.canonicalize_name.
+ * @param name The package name to normalize.
+ * @returns The normalized package name.
+ * @private
+ */
+function canonicalizePackageName(name) {
+  return name.replace(canonicalizeNameRegex, "-").toLowerCase();
+}
```

Comment:
I think `getTransitiveRequirements` already does this so we could leave it out here.

Reply:
I had to keep it to canonicalize the stdlib package names but it now lives in python.js